tests/unit/test_operations.py

```diff
@@ -87,6 +87,16 @@ def test_new_features_identifiers():
     assert set([i['id'] for i in ecoli.notes['changes']['removed']['reactions']]) == set()
     assert is_close(ecoli.optimize().objective_value, ecoli_original.optimize().objective_value)
 
+@pytest.mark.asyncio
+async def test_reactions_change_bounds():
+    ecoli_original = Models.get('iJO1366').copy()
+    ecoli = ecoli_original.copy()
+    ecoli.notes['changes'] = get_empty_changes()
+    reaction_ids = [{'bounds': [0, 1000], 'id': "FBA3"}, {'id': "PFK_3", 'bounds': [-1000, 1000]}]
```

Comment:
Aren't those the default bounds for these reactions? I would make couple of changes: 1) set the bounds to be something more different, like `[1, 3.5]`; 2) test the undo functionality as well.